### ISSUE TITLE
PR not put back to 'needs_reviewer'

### ISSUE THREAD

**Issue by raboof** [timokau/marvin-mk2#99]
In https://github.com/NixOS/nixpkgs/pull/98938#issuecomment-723469357 , the reviewer is inactive, and marvin (correctly) adds a comment that the PR will be set back from `awaiting_reviewer` to `needs_reviewer`, but it doesn't actually do so.

**Comment by timokau** [comment]
It is supposed to set a `timeout_pending` label when posting the warning. For some reason I forgot to add the label (but the code for actually timing out PRs with the label exists). Odd, I'm pretty sure I tested that behavior.

Thanks for the notification!

**Comment by berbiche** [comment]
marvin should probably only consider reviewers with merge access

**Comment by timokau** [comment]
No, making reviews from reviewers without merge access more useful is arguably the main purpose of the bot. See the README of this repo for more info.

**Comment by timokau** [comment]
The nagging message probably could use an overhaul with some more concrete actionable tips for the reviewer, since they might not be familiar with marvin. The usage is already explained in marvin's initial comment, but having the concrete next steps summarized in the nag message would probably help.
